README.md

```diff
@@ -1,3 +1,271 @@
+|GRID_STACKED: Int = 2                                                     |
+
+|Function                                                                            |Description                                 |
+|------------------------------------------------------------------------------------|--------------------------------------------|
+|init(width: Float = 1000, height: Float = 660, numberOfPlots n: Int = 1, numberOfRows nR: Int = 1, numberOfColumns nC: Int = 1, stackPattern: Int = 0)|Initialize a SubPlot |
+|draw(plots: [Plot], renderer: Renderer, fileName: String = "subPlot_output")|Generate plot with the plots passed in as Sub Plots and save the image to disk|
+
+### PlotDimensions
+
+|Function                                                  |Description                                                    |
+|----------------------------------------------------------|---------------------------------------------------------------|
+|init(frameWidth : Float = 1000, frameHeight : Float = 660)|Create a PlotDimensions variable with a frame width and height |
+
+### Point
+
+|Property            |
+|--------------------|
+|x: Float = "X-Axis  |
+|y: Float = "Y-Axis" |
+
+|Function                    |Description                              |
+|----------------------------|-----------------------------------------|
+|init(_ x: Float, _ y: Float)|Create a Point with an x and y coordinate|
+
+### PlotLabel
+
+|Property                          |
+|----------------------------------|
+|xLabel: String = "X-Axis"         |
+|yLabel: String = "Y-Axis"         |
+|labelSize: Float = 10             |
+
+### PlotTitle
+
+|Property                |
+|------------------------|
+|title : String = "TITLE"|
+|titleSize : Float = 15  |
+
+### Color
+
+|Function                                            |Description                                                                |
+|----------------------------------------------------|--------------------------------------------------------------------------|
+|init(_ r: Float, _ g: Float, _ b: Float, _ a: Float)|Create a Color r, g, b and a values. Each of them being between 0.0 and 1.0|
+
+### PlotLabel
+
+|Property                          |
+|----------------------------------|
+|xLabel: String = "X-Axis"         |
+|yLabel: String = "Y-Axis"         |
+|labelSize: Float = 10             |
+
+## Limitations
+- Currently we cannot change the plot dimensions in case the AGGRenderer is used. The plot can be generated with the default dimensions i.e. 1000x660. This issue will be fixed in the future.
+
+## Credits
+1. Maxim Shemanarev : The AGG library is directly used to render plots.
+2. [Lode Vandevenne](https://github.com/lvandeve) : The lodepng library is directly used to encode PNG images.
+3. [Brad Larson](https://github.com/BradLarson) and [Marc Rasi](https://github.com/marcrasi) for their invaluable guidance
+
+## License
+
```

Comment:
I'd recommend moving the License section to be just after the Overview. Licenses are important deciding factors for use of a framework. I did that in my Readme here:

https://github.com/bradlarson/gpuimage2

because it was one of the most common questions I got asked.